src/rust/install.sh

```diff
@@ -186,7 +186,9 @@ else
     curl -sSL --proto '=https' --tlsv1.2 "https://static.rust-lang.org/rustup/dist/${download_architecture}-unknown-linux-gnu/rustup-init" -o /tmp/rustup/target/${download_architecture}-unknown-linux-gnu/release/rustup-init
```

Comment:
Can you bump the patch version for this Feature? Thanks!

https://github.com/devcontainers/features/blob/75602ab7eee94992198fbdb1a3b0d52beb62399a/src/rust/devcontainer-feature.json#L3